pkg/build/controller/build/overrides/overrides_test.go

```diff
@@ -357,3 +358,129 @@ func TestBuildOverrideAnnotations(t *testing.T) {
 		}
 	}
 }
+
+func TestBuildOverrideTolerations(t *testing.T) {
+	tests := []struct {
+		buildTolerations    []corev1.Toleration
+		overrideTolerations []corev1.Toleration
+		expected            []corev1.Toleration
+	}{
+		{
+			buildTolerations:    nil,
+			overrideTolerations: nil,
+			expected:            nil,
+		},
+		{
+			buildTolerations: nil,
+			overrideTolerations: []corev1.Toleration{
+				{
+					Key:   "toleration1",
+					Value: "value1",
```

Comment:
nit - may as well throw in the toleration effect (ex `NoSchedule`).

Reply:
updated in next push